geode-core/src/main/java/org/apache/geode/internal/cache/versions/VersionTag.java

```diff
@@ -350,10 +353,13 @@ public void toData(DataOutput out, boolean includeMember) throws IOException {
     if (this.memberID != null && includeMember) {
       flags |= HAS_MEMBER_ID;
     }
-    if (this.previousMemberID != null) {
+    boolean writePreviousMemberID = false;
```

Comment:
I'm curious why you chose to introduce a new variable rather than referencing the `DUPLICATE_MEMBER_IDS` bit in `flags`?

Reply:
We discussed not duplicating the checks made earlier and I was deeply unsatisfied with redoing checks on "flags".  Just being OCD I guess.

Reply:
ok